eclipse-collections/src/main/java/org/eclipse/collections/impl/bimap/mutable/AbstractMutableBiMap.java

```diff
@@ -823,6 +842,12 @@ public boolean containsAll(Collection<?> source)
             return true;
         }
 
+        @Override
+        public void each(Procedure<? super K> procedure)
+        {
+            AbstractMutableBiMap.this.delegate.keySet().each(procedure);
```

Comment:
@johnjstuart why not directly use `AbstractMutableBiMap.this.forEachKey` here instead of reimplementing something inside this class? It think it would make things simpler and more consistent with the rest of the methods of this class